.golangci.yml

```diff
@@ -19,6 +19,3 @@ linters:
     - goimports
```

Comment:
please replace `deadcode` with `unused` in the list of linters. `deadcode` is deprecated and you should be getting a warning about this when you run golangci-lint locally.

Reply:
Updated accordingly.